app/controllers/projects/project_aliases_controller.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Projects
+  class ProjectAliasesController < ApplicationController
```

Comment:
I think it will make the routes come out as desired if you just call this `Projects::AliasesController` (or `module Projects class AliasesController`). 

For example, the controller for NameDescriptions is `Names::DescriptionsController` even though the model is `NameDescription` (as this is `ProjectAlias`).